help.c

```diff
@@ -474,6 +474,7 @@ static struct cmdnames aliases;
 
 #define AUTOCORRECT_NEVER (-2)
 #define AUTOCORRECT_IMMEDIATELY (-1)
+#define AUTOCORRECT_PROMPT INT_MAX
```

Comment:
This should probably be `-3`, to be consistent with the other special values.

Existing users might have used the numerical value of `INT_MAX` to ask Git to show the options indefinitely but not act on any of them. The current patch would break that scenario.